src/apollo/schema.graphql

```diff
@@ -7,7 +7,7 @@ schema {
 type Query {
   runsList: [Run]!
   runMetadata(runIDs: [ID]!): [Run!]!
-  runTrackingData(runIDs: [ID]!): [TrackingDataset]!
+  runTrackingData(runIDs: [ID]!, showDiff: Boolean): [TrackingDataset]! # showDiff as true will return all tracked fields amongst compared runs - default setting is false
```

Comment:
```suggestion
  runTrackingData(runIDs: [ID]!, showDiff: Boolean = false): [TrackingDataset]! 
```
It would be good to set an explicit default value here for the `showDiff` argument. That way we can safely leave that argument out of this query when we need to.

Reply:
great reminder! Haven't thought of this as I had this as an optional variable in mind but this set up is safer!